reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,476 @@
+          collateralLiquidatedPeriod = collateralLiquidatedPeriod.add(toBN(event.lockedCollateral));
+          uniqueLiquidationsPeriod[event.sponsor] = true;
+        }
+      }
+      allLiquidationStatsTable = {
+        ["# of liquidations"]: {
+          cumulative: Object.keys(uniqueLiquidations).length,
+          [this.periodLabelInHours]: Object.keys(uniqueLiquidationsPeriod).length
+        },
+        ["tokens liquidated"]: {
+          cumulative: this.formatDecimalString(tokensLiquidated),
+          [this.periodLabelInHours]: this.formatDecimalString(tokensLiquidatedPeriod)
+        },
+        ["collateral liquidated"]: {
+          cumulative: this.formatDecimalString(collateralLiquidated),
+          [this.periodLabelInHours]: this.formatDecimalString(collateralLiquidatedPeriod),
+          current: this.formatDecimalString(this.collateralLockedInLiquidations)
+        }
+      };
+
+      console.table(allLiquidationStatsTable);
+    }
+  };
+
+  _generateDisputeStats = async () => {
+    const { toBN } = this.web3.utils;
+
+    let allDisputeStatsTable = {};
+
+    let uniqueDisputes = {};
+    let uniqueDisputesPeriod = {};
+    let tokensDisputed = toBN("0");
+    let tokensDisputedPeriod = toBN("0");
+    let collateralDisputed = toBN("0");
+    let collateralDisputedPeriod = toBN("0");
+    let disputesResolved = {};
+
+    if (this.disputeEvents.length === 0) {
+      console.log(dim("\tNo dispute events found for this EMP."));
+    } else {
+      for (let event of this.disputeEvents) {
+        // Fetch disputed collateral & token amounts from corresponding liquidation event that with same ID and sponsor.
+        const liquidationData = this.liquidationEvents.filter(
+          e => e.liquidationId === event.liquidationId && e.sponsor === event.sponsor
+        );
+        tokensDisputed = tokensDisputed.add(toBN(liquidationData.tokensOutstanding));
+        collateralDisputed = collateralDisputed.add(toBN(liquidationData.lockedCollateral));
+        uniqueDisputes[event.sponsor] = true;
+        if (event.blockNumber >= this.startBlockNumberForPeriod) {
+          tokensDisputedDaily = tokensDisputedPeriod.add(toBN(liquidationData.tokensOutstanding));
+          collateralDisputedDaily = collateralDisputedPeriod.add(toBN(liquidationData.lockedCollateral));
+          uniqueDisputesPeriod[event.sponsor] = true;
+        }
+
+        // Create list of resolved prices for disputed liquidations.
+        const liquidationTimestamp = (await this.web3.eth.getBlock(event.blockNumber)).timestamp;
+        const resolvedPrice = await this.oracleContract.getPrice(
+          await this.empContract.methods.priceIdentifier().call(),
+          liquidationTimestamp,
+          {
+            from: this.empContract.options.address
+          }
+        );
+        // `getPrice` will revert or return the resolved price. `revertWrapper` returns the result if there was no
```

Comment:
nit: are we depending on this bug continuing to exist? Put another way, what happens if a newer version of web3 starts correctly throwing immediately on reverts?

Reply:
Good point. I will re-add the `try-catch` block around this so that it catches an error if it is thrown as expected in a corrected web3